src/UCTSearch.cpp

```diff
@@ -189,18 +189,27 @@ std::string UCTSearch::get_pv(BoardHistory& state, UCTNode& parent) {
     return res;
 }
 
-void UCTSearch::dump_analysis(int playouts) {
-    if (cfg_quiet) {
+void UCTSearch::dump_analysis(int elapsed, bool force_output) {
+    if (cfg_quiet || force_output) {
```

Comment:
Should be `&& !force_output` if we want force_output to mean it should write even in quiet mode?

Reply:
well spotted. Sure.